src/app.py

```diff
@@ -113,7 +113,13 @@ def __add(uuid):
         # Initialize AuthHelper class and ensure singleton
         try:
             if AuthHelper.isInitialized() == False:
-                self.auth_helper_instance = AuthHelper.create(self.APP_CLIENT_ID, self.APP_CLIENT_SECRET)
+                use_sennet_grps = False
+                if hasattr(translator_module.Translator, 'USE_SENNET_GROUPS') and translator_module.Translator.USE_SENNET_GROUPS:
+                    use_sennet_grps = True
+                if use_sennet_grps:
+                    self.auth_helper_instance = AuthHelper.create(self.APP_CLIENT_ID, self.APP_CLIENT_SECRET, use_sennet_grps = True)
```

Comment:
Should be `use_sennet_groups` instead of `use_sennet_grps`
https://github.com/hubmapconsortium/commons/pull/90/files#diff-72e9951a25d6e34af39926348d4ce2ac58f93ca6cb8486988aec1cacc21de5c5R102

Reply:
Good catch @SamSedivy ! I just fixed.